ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -290,6 +294,19 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
                                             {((pref.view === 'tree' || pref.view === 'network') && (
                                                 <Filters pref={pref} tree={tree} resourceNodes={this.state.filteredGraph} onSetFilter={setFilter} onClearFilter={clearFilter}>
                                                     <div className='graph-options-panel'>
+                                                        <a
+                                                            className={`group-nodes-button`}
+                                                            onClick={() => {
+                                                                toggleTitleDirection();
+                                                            }}
+                                                            title={this.state.titleDirection ? 'Title direction right' : 'Title direction left'}>
```

Comment:
I'm not sure if these tooltips are more confusing than helpful. If I hadn't seen the video, the questions these tooltips would raise in my head are: What title? What's a direction in this context?

IMHO, it would be helpful if they are more concise to what will happen, such as "Truncate resource names right" and "Truncate resource names left" - but I'm also not sure on the term "resource names". 

Just as something up for discussion.

Reply:
I'm definitely open to changing it if there's clearer wording, I will switch it to your suggestion for now since it's clearer than what I have at least